.cargo/config

```diff
@@ -3,6 +3,7 @@ xtask = "run --package xtask --"
 
 [target.'cfg(feature = "cargo-clippy")']
 rustflags = [
+    "-Aclippy::needless_borrow", # TODO: remove once msrv increased from 1.48
```

Comment:
This lint started asking to remove `&` from `&[x, y, z]` slices in a lot of places which were incompatible with MSRV :(